addons/medical_statemachine/XEH_postInit.sqf

```diff
@@ -0,0 +1,14 @@
+#include "script_component.hpp"
+
+["ace_killed", { // global event
+    params ["_unit"];
+
+    // Prevent second ragdoll of uncon units when they're killed
+    if (
+        IS_UNCONSCIOUS(_unit) && !isAwake _unit // uncon and not ragdolling
```

Comment:
Funny that the `isAwake _unit` ain't lazy-evaled but `isPlayer _unit` is, like next to all (if not all) others in the mod.